app/assets/stylesheets/welcome.scss

```diff
@@ -31,11 +31,23 @@ $backdrop : #C9D6AE;
   font-size: 80px;
   color: white;
   margin-left: 10px;
-  word-wrap: break-word;
+  width: 560px;
 }
 
 #welcome-description {
   font-size: 24px;
   color: white;
   margin-left: 10px;
+  width: 600px;
+}
+
+#app-info {
+  font-size: 24px;
```

Comment:
Properties should be ordered background, border-radius, color, font-size, margin-left, margin-top, opacity, width